openedx/core/lib/courses.py

```diff
@@ -43,3 +47,32 @@ def create_course_image_thumbnail(course, dimensions):
     _content, thumb_loc = contentstore().generate_thumbnail(course_image, dimensions=dimensions)
 
     return StaticContent.serialize_asset_key_with_slash(thumb_loc)
+
+
+def clean_course_id(model_form, is_required=True):
+    """
+    Validate the course_id for a Django ModelForm.
```

Comment:
Is this comment correct? It seems that it should have the word "clean" in it somewhere. :-)

Should there be a Returns section too?